core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/repair/TestTableRepairJob.java

```diff
@@ -91,9 +91,6 @@
     @Mock
```

Comment:
Couldn't put this at the top of the class, but now we have some unused imports in this test class.

We could also remove myMetadata as it's only used for mocking (not actually used anywhere - and that was the case before as well)